samples/crypto/ecjpake/prj.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_LOG=y
+CONFIG_CONSOLE=y
+CONFIG_PSA_CORE_OBERON=y
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+CONFIG_PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS=y
```

Comment:
`CONFIG_PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS`, what/why is not mentioned in the .rst file.
Should it?

Reply:
Good point, please either add it or create a follow up task for documentation of this symbol.

Reply:
added